### DISCUSSION TITLE
Benchmarks

### DISCUSSION THREAD

**Discussion by vl-dud** [lululxvi/deepxde#1876] [Ideas]
Hi @lululxvi !
I'd like to know if there are existing performance benchmarks of DeepXDE with various backends. This would be relevant for all standard examples, but I am especially interested in operator learning problems. If you have conducted such benchmarks, would you be willing to share them?

**Comment by lululxvi** [comment]
Here is some info I did long time ago https://deepxde.readthedocs.io/en/latest/user/installation.html#which-backend-should-i-choose
But I don't have updated testing.

**Reply by vl-dud** [reply]
This is a little not what I expected, but thank you very much anyway!
